plugins/Marketplace/Api/Client.php

```diff
@@ -249,7 +253,8 @@ private function shouldIgnorePlugin($plugin)
 
     public function searchForThemes($keywords, $query, $sort, $purchaseType)
     {
-        $response = $this->fetch('themes', array('keywords' => $keywords, 'query' => $query, 'sort' => $sort, 'purchase_type' => $purchaseType));
+        $response = $this->fetch('themes',
+          array('keywords' => $keywords, 'query' => $query, 'sort' => $sort, 'purchase_type' => $purchaseType));
```

Comment:
@peterhashair It seems you are applying some automatic code formatting changes here and then. It would be awesome if you could adjust your settings in a way, that the resulting code would have a valid PSR1/4/12 code format. See https://developer.matomo.org/guides/coding-standards